__tests__/app.test.js

```diff
@@ -91,9 +92,50 @@ describe("GET /api/articles", () => {
         expect(typeof article.comment_count).toBe("number");
         expect(article).not.toHaveProperty("body")
       })
-      const dates = body.articles.map((article) => new Date(article.created_at));
-      const formattedDates = [...dates].sort((a, b) => b -a);
-      expect(dates).toEqual(formattedDates);
+      // below can be replaced by Jest Sort
+      // const dates = body.articles.map((article) => new Date(article.created_at));
+      // const formattedDates = [...dates].sort((a, b) => b -a);
+      // expect(dates).toEqual(formattedDates);
     })
   })
-})
\ No newline at end of file
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("status: 200 - responds with all the comments for an requested article", () => {
+    return request(app)
+    .get("/api/articles/1/comments")
+    .expect(200)
+    .then(({body}) => {
+      expect(body.comments).toHaveLength(11);
+      expect(body.comments).toBeSorted("created_at");
+      body.comments.forEach((comment) => {
+        expect(typeof comment.comment_id).toBe("number");
+        expect(typeof comment.votes).toBe("number");
+        expect(typeof comment.created_at).toBe("string");
+        expect(typeof comment.author).toBe("string");
+        expect(typeof comment.body).toBe("string");
+        expect(typeof comment.article_id).toBe("number");
+      });
+      // below can be replaced by Jest Sort.
+      // const dates = body.comments.map((comment) => new Date(comment.created_at));
+      // const formattedDates = [...dates].sort((a, b) => b - a);
+      // expect(dates).toEqual(formattedDates);
```

Comment:
it's great to see your thought process here! However, just remember to remove any comments before pushing code 🙂